leaf-ui/rappid-extensions/ElementInspector.js

```diff
@@ -316,20 +334,20 @@ var ElementInspector = Backbone.View.extend({
     /**
      * Initializes components to display user defined functions
      */
-    renderUserDefined: function(cell){
+    renderUserDefined: function(){
+        
         this.$('#markedValue').hide();
         $(".function-type").val('UD');
 
         // Load the user defined constraints
-        var len = this.intention.getNumOfFuncSegements();
-        var funcSegments = this.intention.dynamicFunction.getFuncSegmentIterable();
-
+        var len = this.model.get('intention').getFuncSegments().length;
```

Comment:
Would it be possible to have a null evolving function at this point? Also - if you switch these lines its a lot less words.
Aka:
var funcSegments = this.model.get('intention').getFuncSegments();
var len = funcSegments.length

Reply:
It shouldn't be possible to have a null evolving function here because this is only called for user defined functions. Also, where it is called is already inside an `if statement` for a null evolving function.

Reply:
Awesome - could you maybe write a comment documenting that somewhere in this function?